woocommerce/woocommerce-admin
# chore: rename WCS to WC Shipping & Tax

### Fixes 

Helps with https://github.com/Automattic/woocommerce-shipping-issues/issues/104
We would like to rename "WooCommerce Services" to "WooCommerce Shipping" and "WooCommerce Tax" (based on context, per https://a8c.slack.com/archives/CU6SYV31A/p1600282268248400 ). See this post for more info: https://wcship.wordpress.com/2020/06/29/research-spike-rename-woocommerce-service/

I renamed all I could find (excluding changelog entries).

### Accessibility



- [n/a] I've tested using only a keyboard (no mouse)
- [n/a] I've tested using a screen reader
- [n/a] All animations respect [`prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)
- [n/a] All text has [at least a 4.5 color contrast with its background](https://webaim.org/resources/contrastchecker/)

### Screenshots

### Detailed test instructions:

- Create store **without** WCS installed
- Set up store for order page prompt to install:
    a. WCS **not** installed
    b. U.S. Based
    c. No extra shipping extensions installed (e.g. 'UPS' live rates)
    d. Add shippable product
- Create order for shippable product
- Go to order edit page
- See shipping banner
![Screen Shot 2020-09-16 at 1 29 33 PM](https://user-images.githubusercontent.com/273592/93377520-b656b880-f820-11ea-92b9-989665825cf8.png)
Other banners:

When setting up new store:
![Screen Shot 2020-09-16 at 1 45 07 PM](https://user-images.githubusercontent.com/273592/93379045-e2733900-f822-11ea-90e9-067fa3e20c76.png)

On the Setup Wizard:
![Screen Shot 2020-09-16 at 1 48 10 PM](https://user-images.githubusercontent.com/273592/93379338-4eee3800-f823-11ea-9d5f-6867dbfee2a9.png)
![Screen Shot 2020-09-16 at 1 48 51 PM](https://user-images.githubusercontent.com/273592/93379388-60cfdb00-f823-11ea-8c83-b2f629a2ed37.png)




### Changelog Note:
